tests/testthat/test-oc_clear_cache.R

```diff
@@ -5,8 +5,8 @@ test_that("oc_clear_cache clears cache", {
   # until a memoise >v.1.1 is released, we need to run oc_get_memoise() twice to
   # have it really cache results
   # https://github.com/ropensci/opencage/pull/87#issuecomment-573573183
-  replicate(2, oc_get_memoise("https://httpbin.org/get"))
```

Comment:
not sure these edits were the right ones. However now one cannot "get" stuff from an URL that's not the OpenCage API, with the refactoring.

Reply:
I am not entirely sure either whether this actually tests what we want to. 
Could we build `oc_get()` in such a way that it accepts a httr2 request object and just performs the request (i.e. just wrapper around `httr2::req_perform()`)? Then we could just build a simple query request here (I'd rather not hit the OpenCage API for this).

Reply:
I am now wondering why we are testing memoization at all, given it's "outsourced" to {memoise}?

Reply:
I'm also not convinced we need to test `oc_clear_cache()` as it only wraps memoise that we trust?

By the way here's another caching method: https://deploy-preview-537--ropensci.netlify.app/blog/2023/04/21/ropensci-news-digest-april-2023/#caching-the-results-of-functions-of-your-r-package